types/sparql-http-client/StreamClient.d.ts

```diff
@@ -53,11 +53,12 @@ declare namespace StreamClient {
         factory?: DataFactory<Q> | undefined;
         Query?: Constructor<TQuery, Q> | undefined;
         Store?: Constructor<TStore, Q> | undefined;
+        maxQuadsPerRequest?: number;
```

Comment:
```suggestion
        maxQuadsPerRequest?: number | undefined;
```
(sticking to used setup if it makes sense)